tests/test_embeds.py

```diff
@@ -11,7 +11,7 @@
 _BASE = {"type": "rich"}
 
 
-@pytest.fixture()
+@pytest.fixture
```

Comment:
BTW, this is configurable, and we can go back to () if its desired

Reply:
Seems fine, as long as it's consistent with other decorators.